src/types/elements/styles.ts

```diff
@@ -35,6 +35,7 @@ type SafeStyle = Pick<CSSProperties, SafeCSSProperty>;
 const CARD_ELEMENT_STYLE_VARIANT_SELECTORS = [
   ':hover',
   ':focus',
+  ':read-only',
```

Comment:
Was this needed for this task? This seems more related to #274?

Reply:
Yes, it is needed for element's #274